cadl-tests/cadl-project.yaml

```diff
@@ -5,5 +5,3 @@ options:
     emitter-output-dir: "{project-root}/cadl-output"
     partial-update: true
     namer: true
-    dev-options:
-      generate-code-model: true
```

Comment:
This still needed. (I say "dev-options.generate-convenience-apis" not needed https://github.com/Azure/autorest.java/blob/5e200a48accd521d95f290ad2f0f72035b1799dc/cadl-extension/src/code-model-builder.ts#L390)